sklearn/utils/tests/test_show_versions.py

```diff
@@ -24,6 +25,7 @@ def test_get_deps_info():
     assert 'pandas' in deps_info
 
 
+@fails_if_pypy
```

Comment:
Fails in `numpy.distutils.system_info.get_info` reported upstream in https://github.com/numpy/numpy/issues/11968

Reply:
This is very strange. I cannot reproduce locally
```
>>>> import sys, numpy; print(numpy.__version__, sys.version)
1.14.3 3.5.3 (fdd60ed87e94, Apr 24 2018, 06:10:04)
[PyPy 6.0.0 with GCC 6.2.0 20160901]
```